module/VuFindSearch/tests/unit-tests/src/VuFindTest/Backend/AbstractHandlerMapTest.php

```diff
@@ -51,7 +51,9 @@ class AbstractHandlerMapTest extends TestCase
      */
     public function testPrepareDefaults()
     {
-        $map = $this->getMockForAbstractClass(AbstractHandlerMap::class);
+        $map = $this->getMockBuilder(AbstractHandlerMap::class)
+                    ->onlyMethods(['getDefaults', 'getAppends', 'getInvariants'])
+                    ->getMock();
```

Comment:
While aligning the `->` operators looks nice, we typically don't include this much extra indentation:

```suggestion
        $map = $this->getMockBuilder(AbstractHandlerMap::class)
            ->onlyMethods(['getDefaults', 'getAppends', 'getInvariants'])
            ->getMock();
```